Explorer/Assets/DCL/InWorldCamera/PhotoDetail/EquippedWearableController.cs

```diff
@@ -71,6 +73,7 @@ async UniTaskVoid AnimateAndAwaitAsync()
         public void Dispose()
         {
             view.wearableBuyButton.onClick.RemoveAllListeners();
+            MarketClicked = null;
```

Comment:
same concern as before

Reply:
in this case this done because the object is managed by `PhotoDetailPoolManager` only and the `Action` that subscribes to that event is given upon creation as a parameter. This is the only place where the event gets cleaned